default.nix

```diff
@@ -1,24 +1,21 @@
-{ pkgs ? import ./nixpkgs.nix
-}:
-
-with pkgs.haskell.lib;
-
 let
-  filterStack = drv: overrideCabal drv (super: {
-     src = builtins.filterSource
-      (path: type: baseNameOf path != ".stack-work")
-      super.src;
-    }
-  );
-  hsPkgs = (import ./stack2nix.nix { inherit pkgs; }).override {
-    overrides = self: super: {
-      cachix = addBuildDepends (generateOptparseApplicativeCompletion "cachix" (justStaticExecutables (filterStack super.cachix))) [ pkgs.boost ];
```

Comment:
Can we enable
 - static
 - optparse applicative completions?

Reply:
You don't need static because haskell.nix builds each component so executable is already statically compiled haskell wise.

I'm not sure optparse applicative completion exists for haskell.nix yet, but it doesn't matter since most folks should have cachix installed via nixpkgs.